lib/web_ui/lib/src/engine/bitmap_canvas.dart

```diff
@@ -172,7 +177,14 @@ class BitmapCanvas extends EngineCanvas {
     _canvasPool.clear();
     final int len = _children.length;
     for (int i = 0; i < len; i++) {
-      _children[i].remove();
+      html.Element child = _children[i];
+      if (child is html.ImageElement) {
```

Comment:
If the image is clipped it won't be a direct child of the root element but a descendant of a clipping element.

Reply:
Done.